From 4a79c77124112de7fd4a4525494031c6955dc2f4 Mon Sep 17 00:00:00 2001
From: Joris Vaillant <joris.vaillant@inria.fr>
Date: Tue, 1 Jul 2025 16:39:02 +0200
Subject: [PATCH] Avoid linking issues when linking against conda package

---
 profiler/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/profiler/CMakeLists.txt b/profiler/CMakeLists.txt
index 9d026b83..0e073b9e 100644
--- a/profiler/CMakeLists.txt
+++ b/profiler/CMakeLists.txt
@@ -184,7 +184,7 @@ else()
 endif()
 
 find_package(Threads REQUIRED)
-target_link_libraries(${PROJECT_NAME} PRIVATE TracyServer TracyImGui Threads::Threads)
+target_link_libraries(${PROJECT_NAME} PRIVATE TracyServer TracyImGui ${CMAKE_DL_LIBS} Threads::Threads)
 
 if(NOT DEFINED GIT_REV)
     set(GIT_REV "HEAD")
-- 
2.49.0

