From cf0a5c15cd392e330521cb4a156bf6abc763ee65 Mon Sep 17 00:00:00 2001
From: Joris Vaillant <joris.vaillant@inria.fr>
Date: Wed, 26 Aug 2026 16:24:33 +0200
Subject: [PATCH] Use find_package instead of package config to avoid issue on
 Windows

---
 cmake/vendor.cmake | 22 ++++++++--------------
 1 file changed, 8 insertions(+), 14 deletions(-)

diff --git a/cmake/vendor.cmake b/cmake/vendor.cmake
index d0083f57..c9c04278 100644
--- a/cmake/vendor.cmake
+++ b/cmake/vendor.cmake
@@ -121,11 +121,9 @@ if(VENDOR_GUI)
     # GLFW
 
     if(NOT USE_WAYLAND AND NOT EMSCRIPTEN)
-        pkg_check_modules(GLFW glfw3)
-        if (GLFW_FOUND AND NOT DOWNLOAD_GLFW)
-            add_library(TracyGlfw3 INTERFACE)
-            target_include_directories(TracyGlfw3 INTERFACE ${GLFW_INCLUDE_DIRS})
-            target_link_libraries(TracyGlfw3 INTERFACE ${GLFW_LINK_LIBRARIES})
+        find_package(glfw3 REQUIRED)
+        if (glfw3_FOUND AND NOT DOWNLOAD_GLFW)
+            add_library(TracyGlfw3 ALIAS glfw)
         else()
             CPMAddPackage(
                 NAME glfw
@@ -145,11 +143,9 @@ if(VENDOR_GUI)
 
     # freetype
 
-    pkg_check_modules(FREETYPE freetype2)
+    find_package(Freetype REQUIRED)
     if (FREETYPE_FOUND AND NOT DOWNLOAD_FREETYPE)
-        add_library(TracyFreetype INTERFACE)
-        target_include_directories(TracyFreetype INTERFACE ${FREETYPE_INCLUDE_DIRS})
-        target_link_libraries(TracyFreetype INTERFACE ${FREETYPE_LINK_LIBRARIES})
+        add_library(TracyFreetype ALIAS Freetype::Freetype)
     else()
         CPMAddPackage(
             NAME freetype
@@ -309,11 +305,9 @@ if(VENDOR_GUI)
 
         # libcurl
 
-        pkg_check_modules(LIBCURL libcurl>=7.87.0)
-        if (LIBCURL_FOUND AND NOT DOWNLOAD_LIBCURL)
-            add_library(TracyLibcurl INTERFACE)
-            target_include_directories(TracyLibcurl INTERFACE ${LIBCURL_INCLUDE_DIRS})
-            target_link_libraries(TracyLibcurl INTERFACE ${LIBCURL_LINK_LIBRARIES})
+        find_package(CURL REQUIRED)
+        if (CURL_FOUND AND NOT DOWNLOAD_LIBCURL)
+            add_library(TracyLibcurl ALIAS CURL::libcurl)
         else()
             CPMAddPackage(
                 NAME libcurl
-- 
2.52.0

