From 5e0188e84b7c38d1d5260c59ee8c72fa78451b43 Mon Sep 17 00:00:00 2001
From: Joris Vaillant <joris.vaillant@inria.fr>
Date: Thu, 9 Jan 2025 16:00:35 +0100
Subject: [PATCH] Add install rule

---
 profiler/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/profiler/CMakeLists.txt b/profiler/CMakeLists.txt
index f9a776c6..a4317bcb 100644
--- a/profiler/CMakeLists.txt
+++ b/profiler/CMakeLists.txt
@@ -173,6 +173,7 @@ endif()
 
 find_package(Threads REQUIRED)
 target_link_libraries(${PROJECT_NAME} PRIVATE TracyServer TracyImGui ${CMAKE_DL_LIBS} Threads::Threads)
+install(TARGETS ${PROJECT_NAME})
 
 if(NOT EMSCRIPTEN)
     target_link_libraries(${PROJECT_NAME} PRIVATE TracyNfd)
-- 
2.34.1

